api/WindowEventHandlers.json

```diff
@@ -371,7 +371,7 @@
               "version_added": null
             },
             "ie": {
-              "version_added": null
+              "version_added": "8"
```

Comment:
Matches https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage#browser_compatibility, so makes sense. Looks like it was a partial implementation back then, but anyway.